pom.xml

```
@@ -324,7 +324,7 @@
     <version.org.jboss.arquillian.extension.drone>2.5.1</version.org.jboss.arquillian.extension.drone>
     <version.org.jboss.arquillian.graphene>2.3.2</version.org.jboss.arquillian.graphene>
     <version.com.google.guava.for-selenium>33.2.1-jre</version.com.google.guava.for-selenium>
-    <version.org.jboss.errai>4.17.0.Final</version.org.jboss.errai>
+    <version.org.jboss.errai>4.16.0.Final</version.org.jboss.errai>
```

```suggestion
    <version.org.jboss.errai>4.17.0.Final</version.org.jboss.errai>
```
This can be removed now.